backend/tsconfig.json

```diff
@@ -10,7 +10,7 @@
         // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
         /* Language and Environment */
         "target": "es2017", /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-        // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+        "lib": ["es2022"],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
```

Comment:
why this change?

Reply:
Typescript was throwing an error when I tried to use `Object.hasOwn()`. The suggested fix was putting this here.